### PR TITLE
Add crates by Copterust

### PR DESCRIPTION
**Pull request by little-arhat**
https://github.com/copterust/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,8 @@ Peripheral Access Crates were also called Device Crates.
 
 ## HAL implementation crates
 
-Implementations of [`embedded-hal`] for microcontroller families and systems running some OS. - ![crates.io](https://img.shields.io/crates/v/embedded-hal.svg)
+Implementations of [`embedded
```
Comment (berkus) [inline README.md]: Hmm, erroneous linebreak here?

File: README.md [hunk 2]
```diff
@@ -478,6 +485,10 @@ Work in progress drivers. Help the authors make these crates awesome!
 [APDS9960]: https://crates.io/crates/apds9960
 [SSD1675]: https://crates.io/crates/ssd1675
 [CC1101]: https://crates.io/crates/cc1101
+[MPU9250_2]: https://crates.io/crates/mpu9250
```
Comment (therealprof) [inline README.md]: Have you made an effort to join forces with the author of the other driver? Not a big fan of confusing people with 2 identical drivers; if this is an evolution of the original driver then I'd prefer replacing the old entry.

CC @japaric

Comment (little-arhat) [reply README.md]: published mpu9250 has more features by now and also battle-tested. japaric/mpu9250 was WIP and wasn't published. So the right solution is to delete japaric/mpu9250 from the list, as it's outdated and causes people to do things like https://github.com/JoshMcguigan/mpu9250/commits/master %)

Comment (therealprof) [reply README.md]: I agree but the right thing to do is to ask @japaric whether it's okay for him, too, before replacing the previous entry.

Comment (little-arhat) [reply README.md]: Sure!

Comment (japaric) [reply README.md]: Since this is a maintained, updated fork I'm OK with removing my old version.

Comment (therealprof) [reply README.md]: Thanks!

Comment (berkus) [reply README.md]: Lets remove the old version in this same PR then, please?

Comment (berkus) [reply README.md]: Make a separate commit removing it and add a note reference to @japaric 's agreement.

**Comment by little-arhat** [comment]
Does it look good now?

File: README.md [hunk 3]
```diff
@@ -422,9 +421,12 @@ Work in progress drivers. Help the authors make these crates awesome!
 - [APDS9960] - I2C - Proximity, ambient light, RGB and gesture sensor - ![crates.io](https://img.shields.io/crates/v/apds9960.svg)
 - [SSD1675] - SPI - Tri-color ePaper display controller - ![crates.io](https://img.shields.io/crates/v/ssd1675.svg)
 - [CC1101] - SPI - Sub-1GHz RF Transceiver - ![crates.io](https://img.shields.io/crates/v/cc1101.svg)
+- [MPU9250] - no_std driver for the MPU9250 (and other MPU* devices) & onboard AK8963 (accelerometer + gyroscope +  magnetometer IMU) ![crates.io](https://img.shields.io/crates/v/mpu9250.svg)
+- [BMP280] - A platform agnostic driver to interface with the BMP280 pressure sensor ![crates.io](https://img.shields.io/crates/v/bmp280-ehal.svg)
+- [LSM303C_2] - A platform agnostic driver to interface with the LSM303C (accelerometer + compass) ![crates.io](https://img.shields.io/crates/v/lsm303c.svg)
```
Comment (therealprof) [inline README.md]:
```suggestion
- [LSM303C] - A platform agnostic driver to interface with the LSM303C (accelerometer + compass) ![crates.io](https://img.shields.io/crates/v/lsm303c.svg)
```

File: README.md [hunk 4]
```diff
@@ -478,6 +480,10 @@ Work in progress drivers. Help the authors make these crates awesome!
 [APDS9960]: https://crates.io/crates/apds9960
 [SSD1675]: https://crates.io/crates/ssd1675
 [CC1101]: https://crates.io/crates/cc1101
+[MPU9250]: https://crates.io/crates/mpu9250
+[BMP280]: https://crates.io/crates/bmp280-ehal
+[LSM303C_2]: https://crates.io/crates/lsm303c
```
Comment (therealprof) [inline README.md]:
```suggestion
[LSM303C]: https://crates.io/crates/lsm303c
```

File: README.md [hunk 5]
```diff
@@ -482,7 +482,7 @@ Work in progress drivers. Help the authors make these crates awesome!
 [CC1101]: https://crates.io/crates/cc1101
 [MPU9250]: https://crates.io/crates/mpu9250
 [BMP280]: https://crates.io/crates/bmp280-ehal
-[LSM303C_2]: https://crates.io/crates/lsm303c
+[LSM303C]: https://crates.io/crates/lsm303c
```
Comment (berkus) [inline README.md]: @therealprof @rahul-thakoor I suggest we sort this section alphabetically for easier navigation and dedupe?

Comment (berkus) [reply README.md]: @little-arhat You changed LSM303C here, but above it is still LSM303C_2, I think they should be in sync.

Comment (therealprof) [reply README.md]: > I suggest we sort this section alphabetically for easier navigation and dedupe?

I agree but let's do that separately.

Comment (berkus) [reply README.md]: Yes, of course separately. Not in this PR.

**Comment by berkus** [comment]
I think it's good now. @therealprof?

**Comment by therealprof** [comment]
Thanks @little-arhat !

**Comment by berkus** [comment]
❤️ Thank you!

**Pull request merged**